ChangeLog.d/issue4286.txt

```diff
@@ -1,11 +1,14 @@
 Removals
-   * Remove the TLS 1.0, TLS 1.1 and DTLS 1.0 support by removing the following
-     library constants: MBEDTLS_SSL_PROTO_TLS1,
-     MBEDTLS_SSL_PROTO_TLS1_1, MBEDTLS_SSL_CBC_RECORD_SPLITTING,
+   * Remove support for TLS 1.0, TLS 1.1 and DTLS 1.0, as well as support for
+     CBC record splitting, fallback SCSV, and the ability to configure
+     ciphersuites per version, which are no longer relevant. This removes the
+     following public constants: MBEDTLS_SSL_PROTO_TLS1,
```

Comment:
Non-blocker: I don't think we need to list every single constant that was removed. The description of the removed features is enough.

Reply:
I kept only the config options then. Could be useful for someone adapting their config scripts or custom file.